README.md

```diff
@@ -1,13 +1,25 @@
-[![CircleCI](https://circleci.com/gh/GSA/datagov-deploy.svg?style=svg)](https://circleci.com/gh/GSA/datagov-deploy)
-
 # data.gov main repository
 
+[![CircleCI](https://circleci.com/gh/GSA/datagov-deploy.svg?style=svg)](https://circleci.com/gh/GSA/datagov-deploy)
```

Comment:
We should probably remove this one and add references to our catalog, inventory and static site deployments.  Just as a complete picture of data.gov.  @btylerburton @Jin-Sun-tts @jbrown-xentity @FuhuXia Thoughts?

Reply:
This is worth removing...

Reply:
Well, the removal was granted.  But I was asking for thoughts on adding badges for all of our live deployments. @btylerburton